cloud/endagaweb/tasks.py

```diff
@@ -439,3 +439,17 @@ def req_bts_log(self, obj, retry_delay=60*10, max_retries=432):
         raise
     finally:
       obj.save()
+
+@app.task(bind=True)
+def block_user(self):
+    """ Block  User if User password is not updated
+    for last 90 days
+    """
+    six_month_ago = (django.utils.timezone.now() -
```

Comment:
`six_month_ago` is not a good name for a variable that might not represent a date exactly six months ago.